.eslintignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
Is this not the default for eslint?

Reply:
just in case. They always got some issues about node_modules folder
https://github.com/eslint/eslint/issues/1163